registry/registry.go

```diff
@@ -41,7 +43,9 @@ func NewHandler(rootfsBlob BlobRef, dropletStore DropletStore, blobs BlobStore)
 }
 
 func Ping(w http.ResponseWriter, r *http.Request) {
-	// 200
+	w.WriteHeader(http.StatusOK)
```

Comment:
Martini is imported by the `cfclient/client.go` code, not by registry. 
The last thing i did after my changes was to updated vendors. So this commit contains all required dependencies for all my changes.

Reply:
ah, we probably should've had the dependencies in an earlier commit, then; ideally every commit should compile and pass tests (so that `git bisect` works). No big deal, will just squash the commits before merging.

Reply:
i think i did that in my last update. I updated dependencies on each commit. It should be better after i reopened the PR.